### PR TITLE
spatial-index: relocate on BBP GitLab

### PR DESCRIPTION
**Pull request by tristan0x**

### CODE REVIEW

File: var/spack/repos/builtin/packages/spatial-index/package.py [hunk 1]
```diff
@@ -7,9 +7,9 @@
 class SpatialIndex(PythonPackage):
     """Spatial indexer for geometries and morphologies"""
 
-    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/hpc/SpatialIndex"
-    git      = "ssh://bbpcode.epfl.ch/hpc/SpatialIndex"
-    url      = "ssh://bbpcode.epfl.ch/hpc/SpatialIndex"
+    homepage = "https://bbpgitlab.epfl.ch/hpc/SpatialIndex"
+    git      = "ssh://git@bbpgitlab.epfl.ch:2201/hpc/SpatialIndex.git"
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/spatial-index/package.py]: 2201 for external use?

**Review by matz-e — DISMISSED** [review]

**Comment by matz-e** [comment]
BTW, I don't expect this to be rebuilt since there's no change in the version, and I'm not certain that the URL goes into the package SHA. (Mental note) Would be good to check that the build was actually triggered.

File: var/spack/repos/builtin/packages/spatial-index/package.py (hunk 1 above)
Comment (tristan0x) [reply var/spack/repos/builtin/packages/spatial-index/package.py]: Now the default SSH port is used, both inside and outside EPFL network \o/

**Comment by matz-e** [comment]
Retest this please

**Comment by tristan0x** [comment]
retest this please Jenkins

**Comment by matz-e** [comment]
Gitlab port move broke part of the CI…

**Comment by matz-e** [comment]
Retest this please

**Comment by matz-e** [comment]
Retest this please

One more time, then I'm going to look in detail

**Review by matz-e — APPROVED** [review]

**Comment by matz-e** [comment]
I think this has been solved otherwise

**Pull request closed without merging**